# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1338477735 14400
#      Thu May 31 11:22:15 2012 -0400
# Branch stable
# Node ID 551566201318bf615b27c60ccf9368f4844008bd
# Parent  42edbe6f6a8dfdf392dca54d0a1fdcb84a515d09
Version 3.6.2 released.

* configure.ac (AC_INIT): Version is now 3.6.2.
(OCTAVE_RELEASE_DATE): Now 2012-05-31.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -22,23 +22,23 @@ dnl Copyright (C) 1993-2012 John W. Eato
 
 ### Preserve CFLAGS and CXXFLAGS from the environment before doing
 ### anything else because we don't know which macros might call
 ### AC_PROG_CC or AC_PROG_CXX.
 
 EXTERN_CFLAGS="$CFLAGS"
 EXTERN_CXXFLAGS="$CXXFLAGS"
 
-AC_INIT([GNU Octave], [3.6.2-rc2], [http://octave.org/bugs.html], [octave])
+AC_INIT([GNU Octave], [3.6.2], [http://octave.org/bugs.html], [octave])
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION arg
 OCTAVE_VERSION="$PACKAGE_VERSION"
 OCTAVE_API_VERSION_NUMBER="48"
 OCTAVE_API_VERSION="api-v$OCTAVE_API_VERSION_NUMBER+"
-OCTAVE_RELEASE_DATE="2012-05-24"
+OCTAVE_RELEASE_DATE="2012-05-31"
 OCTAVE_COPYRIGHT="Copyright (C) 2012 John W. Eaton and others."
 AC_SUBST(OCTAVE_VERSION)
 AC_SUBST(OCTAVE_API_VERSION_NUMBER)
 AC_SUBST(OCTAVE_API_VERSION)
 AC_SUBST(OCTAVE_RELEASE_DATE)
 AC_SUBST(OCTAVE_COPYRIGHT)
 
 AC_REVISION($Revision: 1.603 $)
